src/include/brain/workload/workload_defaults.h

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// workload_defaults.h
+//
+// Identification: src/include/brain/workload/workload_defaults.h
+//
+// Copyright (c) 2015-2018, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+/**
+ * This header file contains default attributes
+ * associated with the workload prediction task
+**/
+
+namespace peloton {
+namespace brain {
+
+struct WorkloadDefaults {
```

Comment:
I think this should be called LSTMDefaults. We'll have other default values for other models. We can keep the file name workload defaults.